src/chainstate/stacks/index/marf.rs

```diff
@@ -90,10 +88,30 @@ pub struct MARF {
 
 struct WriteChainTip {
     block_hash: BlockHeaderHash,
-    height: u64
+    height: u32
+}
+
+#[cfg(not(test))]
+fn short_circuit_genesis_test(_s: &TrieFileStorage) -> Option<BlockHeaderHash> {
+    None
```

Comment:
Can you leave a `TODO` that this will need to be filled in at some point?

Reply:
I don't think it should ever be filled in. The "short_circuit_genesis_test" is only used in testing the Trie structure without the MARF. Because the MARF is what fills in the block height/block hash map, the trie tests would get really unhappy without that mapping in place. This function allows tests to short circuit those lookups.

I can change this to a conditional code block, though, which didn't used to be possible in Rust.

Reply:
Ah, gotcha.  No worries then, just as long as it's explained as a comment.

Reply:
Comments added in d31f0207